blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: SourceCred
+author_url: https://github.com/sourcecred
+author_image_url: https://avatars3.githubusercontent.com/u/35711667?s=200&v=4
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, this post only touches on several major additions
+here.
+
+### Discourse plugin
+
+As of this release, you can use SourceCred to analyze [Discourse] forums
+(for example, SourceCred's [own forum]). Our experience so far is that it's
+one of the most interesting plugins, and it works well out of the box!
+
+The basic logic of the Discourse plugin is that every topic, post, and like
+on the forums can flow cred, with cred accumulating at posts that received
+lots of likes, or lots of high-cred responses. Cred also flows between posts
+that reference each other. And the users that authored high cred posts wind up
+with plenty of cred themselves.
+
+The plugin doesn't require any special credentials, so you can easily explore
+Discourse forums you're curious about using SourceCred. It can give a
+perspective on which people are adding content most valued by the community.
+The [plugin setup guide][discourse setup] walks through the commands to do
+this.
+
+### Initiatives
+
+[Initiatives] are added to SourceCred by a new plugin of the same name.
+Instructions are in its [setup guide][initiatives setup].
+
+The goal of the initiatives plugin is to allow tracking important work that
+wasn't directly captured on existing platforms like GitHub or Discourse. You
+can think of the initiatives plugin as being a "manual mode" way to tune the
+contribution graph. Within SourceCred itself, we're using initiatives to track
+work like organizing community calls, championing higher-level features, or
+supporting our partner projects.
+
+The plugin is still in an early form--consider it an alpha-level feature.
+Currently, adding initiatives requires manually tweaking json files on GitHub,
+but we're hoping to make this more intuitive and accessible by adding a UI in
+the near future.
```

Comment:
(Want to remove this, but commenting here before I do so you can comment on it.)
I think the release notes aren't a great place to promise future features.

Because blogs are not a live document to be updated as timelines change, and it might create the assumption we don't need help with it. As publicly committing to it seems like we've got this under control and people just need to sit back and wait for it to arrive.

Reply:
Sure, I'm fine with that getting removed.